File: WSViajes/WSViajes/Models/Request/LoginRequest.cs
namespace WSViajes.Models.Request
{
    public class LoginRequest : Respuesta
    {
        public string Email { get; set; }
        public string Password { get; set; }
        public byte TipoUsuario { get; set; }

        public LoginRequest()
        {
            this.Email = string.Empty;
            this.Password = string.Empty;
        }
    }
}

File: Administrador/src/app/classes/request/LoginRequest.ts
export class LoginRequest {
  public Email: string;
  public Password: string;
  public TipoUsuario: number;
  constructor( object: any){
    this.Email = (object.Email) ? object.Email : null;
    this.Password = (object.Password) ? object.Password : null;
    this.TipoUsuario = (object.TipoUsuario) ? object.TipoUsuario : null;
  }
}
